### PR TITLE
Reduce detection strength of guard monsters

### PR DESCRIPTION
**Pull request by dbenage-cx**
Guard monster vision changes regarding #537.

For Release milestone, in the event new parts are not polished in time.

### CODE REVIEW

File: default/scripting/ship_hulls/monster/SH_IMMOBILE_FACTOR.focs.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ Hull
     Unproducible
     location = All
     effectsgroups = [
-        [[GOOD_VISION]]
+        [[UNOWNED_OWNED_VISION_BONUS(WEAK,15,10)]]
```
Comment (MatGB) [inline default/scripting/ship_hulls/monster/SH_IMMOBILE_FACTOR.focs.txt]: Can't have value 15, detection strengths need to always be a multiple of 10, stealth values then fit between them (normally ending in 5) to ensure there's never doubt about whether something is higher or lower.

Comment (dbenage-cx) [inline default/scripting/ship_hulls/monster/SH_IMMOBILE_FACTOR.focs.txt]: rebased.  dropped to 10 here and raised to 40 for Acirema/Sentinel (catching +stealth organics without cloak)

**Comment by MatGB** [comment]
Sensible, merged in, thanks—I do of course reserve the right to change all the numbers when I get around to rebalancing the whole thing but let's face it that'll be awhile ;-)

**Comment by Vezzra** [comment]
Assuming this addresses #537 adequately I'm going to close #537.

**Comment by MatGB** [comment]
Knew I'd forgotten to do something. Yup, does what was needed and then some, will be easier to change moving forward as well, which is nice.

**Pull request merged**